src/debugger/nodeDebugWrapper.ts

```diff
@@ -158,6 +160,7 @@ new EntryPointHandler(ProcessType.Debugger).runApp(appName, () => version,
             args.args = [
                 args.platform,
                 debugServerListeningPort.toString(),
+                args.iosRelativeProjectPath || IOSPlatform.DEFAULT_IOS_PROJECT_RELATIVE_PATH,

```

Comment:
If you specify on purpose iosRelativeProjectPath: "" this will use "ios" instead.